### PR TITLE
New meters

### PR DESCRIPTION
**Pull request by SzymonPobiega**
An alternative version of #6719 on top of #6718. I hope I didn't lose any change.

### CODE REVIEW

File: Snippets/Core/Core_9.1/Core_9.1.csproj [hunk 1]
```diff
@@ -20,4 +20,4 @@
     <PackageReference Include="OpenTelemetry" Version="1.3.0" />
```
Comment (lailabougria) [inline Snippets/Core/Core_9.1/Core_9.1.csproj:20]: The other PR has solution file changes that don't appear in this one, is that intentional?

File: Snippets/Core/Core_9.1/Core_9.1.csproj [hunk 2]
```diff
@@ -20,4 +20,4 @@
...
   </ItemGroup>
 
-</Project>
\ No newline at end of file
+</Project>
```
Comment (lailabougria) [inline Snippets/Core/Core_9.1/Core_9.1.csproj:23]: The changes here also don't match: https://github.com/Particular/docs.particular.net/blob/bfff555f06651bc7b8e1a72622c41ca514e1c075/Snippets/Core/Core_9.1/Core_9.1.csproj

File: nservicebus/operations/opentelemetry_host-identifier_core_[8,).partial.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ Guid serviceInstanceId = GuidUtility.Create(new Guid("4d63009a-8d0f-11ee-aad7-4c
 
 // OpenTelemetry
 services.AddOpenTelemetry()
-    .ConfigureResource(rb => rb.AddService("MyService", serviceInstanceId: serviceInstanceId.ToString()))
+    .ConfigureResource(builder => builder.AddService("MyService", serviceInstanceId: serviceInstanceId.ToString()))
```
Comment (lailabougria) [inline nservicebus/operations/opentelemetry_host-identifier_core_[8,).partial.md:25]: We may want to address David's comment on this part to make this a snippet

File: nservicebus/operations/opentelemetry_metrics_core_[9.1,).partial.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+## Meters
+
+NServiceBus endpoints can be configured to expose metrics related to message processing. To capture meter information, add the `NServiceBus.Core` meter source to the OpenTelemetry configuration:
+
```
Comment (SzymonPobiega) [inline nservicebus/operations/opentelemetry_metrics_core_[9.1,).partial.md:4]:
```suggestion
NOTE: The metric definitions published by NServiceBus are not yet finalized and could change in a minor release.

```

File: nservicebus/operations/opentelemetry_host-identifier_core_[8,).partial.md (hunk 3 above)
Comment (SzymonPobiega) [reply nservicebus/operations/opentelemetry_host-identifier_core_[8,).partial.md]: Fixed.

**Pull request merged**